### PR TITLE
Remove EShield from Traitor uplink, Nerf EShield price in Nukie uplink.

### PR DESCRIPTION
**Pull request by LordEclipse**
## About the PR
EShields are a powerful item, and as such their price is being adjusted
## Why / Balance
Its a very good item, nukies shouldn't be able to trivially buy a ton of them for something they get a lot of mileage out of.

**Changelog**

 - tweak: The Syndicate is having Energy Shield shortages, the price of Energy Shields in the Nuclear Operative Uplink have gone from 6 Telecrystals, to 8 Telecrystals.
 
 - remove: The Syndicate can no longer supply their Traitors with Energy Shields. Energy Shields have been removed from Traitor Uplinks.

### CODE REVIEW

**Comment by LordEclipse** [comment]
> 12 tc for a defensive option is a bit much, considering the energy sword is 8 tc. If you REALLY feel the shield needs a nerf, maybe increasing it 8 would be better, the noslips don't need a change since nukies shouldn't be able to be countered by a bar of soap under any circumstances.

Objectively better options are not good.

**Comment by Kadeo64** [comment]
> 12 tc for a defensive option is a bit much, considering the energy sword is 8 tc. If you REALLY feel the shield needs a nerf, maybe increasing it 8 would be better

the energy shield completely counters all lasers. all. lasers. the most common type of weapon, completely countered. 12tc makes the energy shield from something nukies can buy and be set for the rest of the ops to something they actively have to think about. Even 8tc I think is too low to make energy shield's usage drop.

**Comment by deltanedas** [comment]
keep noslips at 2tc it hurts traitor too much imo

**Comment by LordEclipse** [comment]
> keep noslips at 2tc it hurts traitor too much imo

Yeah I had to create 2 separate listings, 12 TC eshield was too much to be worth it for tators, so I instead made an 8 TC version for traitors, and a 12 TC for Nukies. And noslips are staying 2 TC for traitors, but being bumped up to 4 TC for nukies.

**Comment by GreyMario** [comment]
how about you train players on how to shoot bullet based weaponry at things that counter lasers

**Comment by LordEclipse** [comment]
> how about you train players on how to shoot bullet based weaponry at things that counter lasers

A nukie can almost survive a full WT550 clip being hit by every single shot, they just barely got downed. A completely unarmored person doesn't even have 25 damage after being shot with 10 shots from the captain laser gun. 




https://github.com/space-wizards/space-station-14/assets/106132477/95065900-4dc2-4adb-b221-058f5d6862fa

image

image

**Comment by Rainbeon** [comment]
Why should no-slips be 4 TC when blood-red mags are also 4 TC and overall superior?

**Comment by asperger-sind** [comment]
> Why should no-slips be 4 TC when blood-red mags are also 4 TC and overall superior?

They're more of a sidegrade, you get to move around in nograv as well as a jetpack but:

1. Valid Salad
2. 10% movespeed penalty when used
3. No-slip effect applies only when used

**Comment by Kadeo64** [comment]
> They're more of a sidegrade, you get to move around in nograv as well as a jetpack but:
> 
> 1. Valid Salad
> 2. 10% movespeed penalty when used
> 3. No-slip effect applies only when used

only 5% so negligible

**Comment by asperger-sind** [comment]
> only 5% so negligible

Pretty sure it's 5% on walking and 10% on running, checked the .yml before commenting.
Even then, 10% can still make a difference, especially if it adds up with the duffel and hardsuit.

**Comment by UbaserB** [comment]
Noslips is at a perfect area to purchase as a syndicate, and any changes would like mess that up

**Comment by LordEclipse** [comment]
> Noslips is at a perfect area to purchase as a syndicate, and any changes would like mess that up

They are in a separate listing for nukie and traitor, Nukies have noslips at 4 TC, and Traitors have them still at 2 TC.

**Comment by Ilya246** [comment]
the noslip nerf just hurts stealthops, pretty sure non-stealth nukies will now always just buy redboots since you can kill grav and gain a massive advantage with them and you get a free emergency jetpack
should maybe make noslips 3 TC instead

**Comment by ps3moira** [comment]
I think it's at a good level where it's at, crew shouldn't be able to take down Nukies so easily and nerfing this will make Nukies have an even harder time. Stun/Slip shouldn't be the meta against nukies, ACTUAL COMBAT should be the meta

**Comment by K-Dynamic** [comment]
As someone who always wanted noslip and eshield nerfs, noslips should be between 3 to 4 tc (if secoff slipping is just skill issue, maybe syndies should get a taste too and simply walk lol), eshield stats decreased and cost from 6 tc to between 8 to 10 tc

**Comment by K-Dynamic** [comment]
> > how about you train players on how to shoot bullet based weaponry at things that counter lasers
> 
> A nukie can almost survive a full WT550 clip being hit by every single shot, they just barely got downed with 28 shots (if I counted correctly). A completely unarmored person doesn't even have 25 damage after being shot with 10 shots from the captain laser gun. This is ignoring the fact that players in game will also attempt to evade being shot at or duck behind cover.
> 
>  2024-01-25.20-07-26.mp4 
> image
> 
> image

Also keep in mind that eshields are like 200 hp, crew usually only get lasers when armoury is blasted (which shields ignore), and that damage to a nukie is easily healable via omnizine and puncturase

Hence nerf needed

**Comment by magmodius** [comment]
> > Noslips is at a perfect area to purchase as a syndicate, and any changes would like mess that up
> 
> They are in a separate listing for nukie and traitor, Nukies have noslips at 4 TC, and Traitors have them still at 2 TC.

personally think these should be swapped (4TC for tots and 2TC for nukies) as this keeps the bloodred magboots effective as an upgrade for more TC while handicapping traitors who brainlessly buy noslips every time they go loud and just break welding tanks and throw soaps at everyone.
no syndie item should be a complete crutch to a playstyle. it's why the rigged boxing gloves were nerfed so hard, and why nocturine is now near impossible to get for a non chemist. if noslips are being bought every time a syndicate agent wants to go loud, then that means they're either too low price or too powerful, neither of which are fun for the poor station victims who have to deal with it. 

tl;dr is that metas aren't fun and i genuinely believe that a loud syndie using noslips has far too many options to fuck over everyone else fighting them for it to be only 2TC. bumping them to 4TC is valid.

**Comment by Ilya246** [comment]
i would recommend to just not touch noslips and make eshield 8 TC and 150 HP so it's the same HP as a bloodred, plus remove the absurd laser resist and replace it with a smaller one

**Comment by K-Dynamic** [comment]
I have no problem with nukies having bloodred boots or noslips, in fact they should be roundstart for them in their armoury - we currently give them visors and flash/weldproof mask so they can do their job as an elite strike force that can't just be slipped.

Traitors however... noslips for traitors let you avoid avoid a rather painful and somewhat underexplained mechanic, which the rest of the station can't without galoshes or magboots. However it also has the side effect of letting you murderbone if you just splash lube and oil everywhere for just 2 tc, which doesn't make much sense for something meant to be a stealthy gadget

**Comment by asperger-sind** [comment]
>as this keeps the bloodred magboots effective as an upgrade for more TC
 
 As I've said earlier, magboots are more of a sidegrade, you can argue that as a sidegrade they outperform noslips in most scenarios, but if you don't use the features of the magboots they're worse than noslips; and you most likely won't use them.
 
 > while handicapping traitors who brainlessly buy noslips every time they go loud and just break welding tanks and throw soaps at everyone
 
This won't handicap the occasional loud traitor; they can just murder one of the janitors at this rate. Neither is this strategy a bad thing to have because it comes off with a tradeoff and has its counters.

>no syndie item should be a complete crutch to a playstyle.

This isn't true, if you want to be a pseudo-nukie you need a blood-red hardsuit. Want to be a comms agent? You need a voice changer. Want to do crazy perma breakouts? You need an uplink implanter, etc. Uplink items can and at times should be a key to certain playstyles.

>it's why the rigged boxing gloves were nerfed so hard, and why nocturine is now near impossible to get for a non chemist.

The rigged glove nerf is mild, I nerfed them to not be too unfair on the receiving end while keeping them a decent item nonetheless, you buy nocturine in the uplink without the chem kit.

> if noslips are being bought every time a syndicate agent wants to go loud, then that means they're either too low price or too powerful, neither of which are fun for the poor station victims who have to deal with it.

No-slips aren't bought because they are a necessity for loud traitors, but because it prevents the rare event of the traitor slipping and dying. No-slips are a strong item because they are cheap and powerful at the same time, if the price was to go up the popularity of no-slips would greatly diminish because one of the factors isn't met.


> tl;dr is that metas aren't fun and i genuinely believe that a loud syndie using noslips has far too many options to fuck over everyone else fighting them

Just as the crew has the option to not dance to the traitor's tune and walk into the puddle of oil and lube, or to screw them over completely with incendiary shells and the like. Loud syndies are on the crew's ground and if the crew decides to not use this advantage and go headfirst into a loud traitor knowing they most likely won't win, then they probably shouldn't win in the first place.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -1343,9 +1363,30 @@
...
+  productEntity: EnergyShield
+  cost:
+    Telecrystal: 12
+  categories:
+  - UplinkMisc
+  conditions:
+  - !type:StoreWhitelistCondition
+    whitelist:
+      tags:
+      - NukeOpsUplink
```
Comment (keronshb) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1389]: Traitors shouldn't have access to the regular EShield and it should just be upped to 8 TC in general.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 2]
```diff
@@ -1221,6 +1221,26 @@
...
+
+- type: listing
+  id: UplinkClothingNoSlipsShoesNukie
+  name: uplink-clothing-no-slips-shoes-name
+  description: uplink-clothing-no-slips-shoes-desc
+  productEntity: ClothingShoesChameleonNoSlips
+  cost:
+    Telecrystal: 4
```
Comment (keronshb) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1236]:
```suggestion
  cost:
    Telecrystal: 2
```

Noslips are fine at 2tc, I don't really see an issue with them.

**Review by keronshb — CHANGES_REQUESTED** [review]
Personally I'd say get rid of E Shields for traitors entirely, they should be available for Nukies only. And Noslips are a bit too expensive at 4tc.

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 1 above)
Comment (LordEclipse) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:1389]: There does need to be a price nerf, but its an item that has 200 HP, and high reflection chance, I'm not sure 8 TC is high enough.

Comment (keronshb) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:1389]: It's meant to have high HP since you're spending money and can't just replace or reprint it. Higher than 8 starts to get into territory of "is it really worth buying this consumable vs the HP cost". Maybe we try 8tc first, and if it's still too little we do another increment to 10. Then ultimately 12 if it's still too little.

**Comment by keronshb** [comment]
Oops removed freeze, thought this was still frozen but it was put into a different category #24346

File: Resources/Prototypes/Catalog/uplink_catalog.yml (hunk 1 above)
Comment (LordEclipse) [reply Resources/Prototypes/Catalog/uplink_catalog.yml:1389]: Yeah it can always be adjusted. Sure.

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 3]
```diff
@@ -1337,15 +1337,20 @@
   - UplinkMisc
 
 - type: listing
-  id: UplinkEshield
+  id: UplinkEshieldNukie
```
Comment (keronshb) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:1340]: No need to change the name to Nukie, the tag takes are of that.

**Review by keronshb — CHANGES_REQUESTED** [review]

**Review by keronshb — APPROVED** [review]

**Comment by Varen** [comment]
How did this turn from nukies balance to removing an iconic item from the traitor uplink? What justification is there? I can't see anything in the discussion that even touches on traitor use of it, only nukie. Nukie gameplay is a completely different gamemode to traitor, that's the point of having the option of separate listings.

**Comment by Varen** [comment]
> > How did this turn from nukies balance to removing an iconic item from the traitor uplink? What justification is there? I can't see anything in the discussion that even touches on traitor use of it, only nukie. Nukie gameplay is a completely different gamemode to traitor, that's the point of having the option of separate listings.
> 
> traitor isn't supposed to be a powerful force that can kill the entire station. The shield doubles your EHP and negates lasers, which is good for nukies who are taking blow after blow and can make good use of something to beef them up a bit, but traitors don't need to wage war on security unless they REEEAAALLLY screwed up.

I'd love to see a replay of this issue having happened on a live server.
